lib/spack/llnl/util/tty/__init__.py

```diff
@@ -233,7 +233,7 @@ def hline(label=None, **kwargs):
     out.write(label)
     out.write(suffix)
 
-    print out.getvalue()
+    print(out.getvalue())
```

Comment:
There are no substantive changes in this file.  Although I agree with them, I believe PRs are better if they present the minimum changes required for the feature at hand.  Can you move these changes to a different PR?